utils/auth.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""
+Place to store auth-related code, e.g. for dealing with certs or session tokens.
+"""
+import logging
+import os
+
+from utils.base import ImplementationError, MilMoveEnv, is_local
+from utils.constants import DP3_CERT_KEY_PEM
+
+logger = logging.getLogger(__name__)
+
+
+def set_up_certs(env: MilMoveEnv) -> None:
+    """
+    Sets up certs for making requests to the mymove server
+    :param env: MilMoveEnv that the target server is running in, e.g. MilMoveEnv.DP3
+    :return: None
+    """
+    if is_local(env=env):
+        return  # We don't need to set up certs for a local run because they already exist
+
+    host_upper = env.value.upper()
+
+    try:
+        deployed_tls_cert = os.environ[f"MOVE_MIL_{host_upper}_TLS_CERT"]
+        deployed_tls_key = os.environ[f"MOVE_MIL_{host_upper}_TLS_KEY"]
+    except KeyError:
+        logger.debug(f"Unable to find cert and key values for environment: {MilMoveEnv}")
+
+        raise ImplementationError(
+            "Cannot run load testing in a deployed environment without the matching certificate and key."
+        ) from None
+
+    with open(DP3_CERT_KEY_PEM, "w") as f:
```

Comment:
Does hardcoding this prevent us from running against say experimental?

Reply:
To be honest, I'm not sure. In the past I would have said yes, but since experimental got moved to `dp3`, it might be that our cert key might work with either one. But it would require us to edit other code like the bit about which base domain to point to.

I wasn't too concerned since my understanding is that we really only want to run against the load testing environment. We only used experimental before because we didn't have a dedicated environment. Do you think we should keep the ability to point at other environments though?

Reply:
Oh I had forgotten that exp moved outside of the ATO boundary, I was mainly thinking we'd want to know if load testing performs differently inside the ATO infra.

It's a bit weird that we expose the host field in locust as configurable/overideable but make slightly different assumptions here.

No action needed.

Reply:
Yeah that is one weird change this introduces in that we don't really use the locust host for much other than as a starting point. Normally the locust host is used in the background by the locust client to create the full URL, but these changes make it so that the host is just a base string that we start off with and then we pass the client full urls that our `MilMoveRequestPreparer` sets up